finos/legend-engine
# Support for query by dates & result parsing

#### What type of PR is this?
- Improvement
#### What does this PR do / why is it needed ?

- Support for query filters  with dates.
- MongoDocumentFormat internalize now supports date formats returned by Mongo (as ISO dates instead of string in json)
- Test for service compilation
- Make identity attribute available to connection provider class, so it can be used in conjunction with Kerberos authentication.
- Add grammar & protocol dependency for Legend studio to be able to  compile Mongo Database store/mapping/runtime DSL
- Move test-binder to executionPlan-test module so that other core modules do not see this during build pipeline.
